Tasks/ContainerStructureTestV0/testrunner.ts

```diff
@@ -50,30 +59,30 @@ export class TestRunner {
                 return null;
         }
     }
-    
-    private async downloadTestRunner(downloadUrl: string): Promise<string> {
-        const gcst = path.join(__dirname, "container-structure-test");
-        return downloadutility.download(downloadUrl, gcst, false, true).then((res) => {
-            chmodSync(gcst, "777");
-            if (!existsSync(gcst)) {
-                tl.error(tl.loc('FileNotFoundException', path));
-                throw new Error(tl.loc('FileNotFoundException', path));
-            }
-            return gcst;
-        }).catch((reason) => {
-            tl.error(tl.loc('DownloadException', reason));
-            throw new Error(tl.loc('DownloadException', reason));
-        })
-    }
 
     private runContainerStructureTest(runnerPath: string, testFilePath: string, image: string): string {
-        let command = tl.tool(runnerPath);
-        command.arg(["test", "--image", image, "--config", testFilePath, "--json"]);
-    
-        const output = command.execSync();
+        existsSync(runnerPath);
```

Comment:
These two should be done where you downloaded the runner exe. This function should not set any permissions.

Reply:
Makes sense, will change it.